README.md

```diff
@@ -95,6 +95,24 @@ The password argument is optional. It's required only if your system is unable t
 
 ```python install_all.py --password xyz```
 
+
+## Docker environment execution
+
+A docker environment initially created in order to compile and execute snakeskii21 in any system, as follows:
+
+1. Get into `docker_env` directory and build it executing:
+	`docker build -t snake_env .`
+2. After it's build, run container as follows (newbie's gentle execution):
+	`docker run -it --rm -v <snakeskii21_path>:/data -w /data snake_env /bin/bash`
+
+	Rationale:
+		`-it` stands for `iterative` mode (-i keeps STDIN attached and open, while -t allocate a pseudo-tty)
+		`--rm` cleans container up after execution (for small tests this flag is way useful to ignore)
+		`-v HOST_DIR:CONTAINER_DIR` sets a host directory share point inside the container
+		`-w WORKDIR_NAME` is your initial working directory after bring it up
```

Comment:
Since we already explain this in the [Docker README](/docker_env/README.md) (this link will be broken when I make the suggestion but it'll fix itself once the PR is merged (I hope)) it's a good idea to not repeat ourselves and just link to that document (which also has a lot more information).
```suggestion
For instructions on how to compile and execute Snaskii21 using Docker, check the [Docker README](/docker_env/README.md).
```